packages/@okta/vuepress-site/docs/guides/anything-as-a-source/index.md

```diff
@@ -0,0 +1,374 @@
+    * Set `entityName` to `USERS`. Only user import is supported; group import is not supported.
+    * If you need to add more users, make another [Load user data profile](/docs/reference/api/xaas) request with the same `sessionId` value. You can make up to 50 load user data requests for one Import Session.
+
+    ```bash
+    curl -i -X POST \
+      'https://${yourOktaDomain}/api/v1/identity-sources/${identitySourceId}/sessions/${sessionId}/bulk-upsert' \
+    -H 'Authorization: SSWS ${apiKey}' \
+    -H 'Content-Type: application/json' \
+    -d '{
+        "entityName": "USERS",
+        "profiles": [
+            {
+                "externalId": "${userId}",
+                "profile": {
+                    "userName": "${userName}",
+                    "firstName": "${firstName}",
+                    "lastName": "${lastName}",
+                    "email": "${userEmail}",
+                    "secondEmail": "${userEmail2}",
+                    "mobilePhone": “${userMobileNumber}",
+                    "homeAddress": "${userAddress}"
+                }
+            }
+        ]
+    }'
+    ```
+
+    Possible returned responses:
+    * **202 ???** : The bulk import was successful. A `batchId` value is returned.
+    * **400 Bad Request**: Another active Import Session exists for the same identity source.
+    * **403 Access Denied**
+
+3. [Trigger the data import process](/docs/reference/api/xaas):
+
+    Once you have imported all your user profiles to insert or update, start the import processing job to add users to the Okta Universal Directory:
+
+    ```bash
+    curl -i -X PUT \
+      'https://${yourOktaDomain}/api/v1/identity-sources/${identitySourceId}/sessions/${sessionId}/start-import' \
+    -H 'Authorization: SSWS ${apiKey}' \
+    -H 'Content-Type: application/json' 
+    ```
+
+    Possible returned responses:
+    * **200 OK**: The data import process started successfully and returns the following properties:
+
+        ```json
+          {
+            "id": "${sessionId}",
+            "identitySourceId": "${identitySourceId}",
+            "status": "IN_PROGRESS"
+        }
+        ```
+
+    * **400 Bad Request**: Another active Import Session exists for the same identity source.
+    * **403 Access Denied**
+
+4. Monitor the Import Session until the data processing completes
+
+    Request for status of the Import Session to verify that the import job is complete:
+
+    ```bash
+    curl -i -X GET \
+      'https://${yourOktaDomain}/api/v1/identity-sources/${identitySourceId}/sessions/${sessionId} \
+    -H 'Authorization: SSWS ${apiKey}' \
+    -H 'Content-Type: application/json' 
+    ```
+
+    Alternatively, use the [Import Monitoring](https://help.okta.com/okta_help.htm?id=ext-view-import-monitoring-dashboard) page in the Admin Console.
+
+    Possible returned responses:
+    * **200 OK**: The data import process started successfully and returns the following properties:
+
+        ```json
```

Comment:
We can refine this further based on the latest state of the session.
It could be in CREATED or IN_PROGRESS or in COMPLETED status under non-error flow.